backend/python/app/graphql/types/entity_type.py

```diff
@@ -0,0 +1,35 @@
+from ...models import Entity as EntityModel
+from ...resources.entity_dto import EntityDTO
+
+import graphene
+from graphene import relay
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+class Entity(SQLAlchemyObjectType):
+    class Meta:
+        model = EntityModel
+        interfaces = (relay.Node, )
+
+class EntityInput(graphene.InputObjectType):
+    string_field = graphene.String(required=True)
+    int_field = graphene.Int(required=True)
+    string_array_field = graphene.List(graphene.String, required=True)
+    enum_field = graphene.String(required=True) # TODO: make stricter
+    bool_field = graphene.Boolean(required=True)
+
+def entity_input_to_dto_and_model(entity_data):
```

Comment:
A summary of Kristy's comment which I accidentally deleted: Not sure if this is how it should be implemented. Currently doing it this way since typescript wouldn't work without an id. Starter code has `EntityDTORequest` and `EntityDTOResponse` to solve this issue, but it might be redundant for us since we already have `Entity (model)`

Reply:
In the past I've just interacted directly with the model instead of using a DTO with GraphQL and based on what I read in a quick google search of "GraphQL DTO" it sounds like the two design patterns go against each other.

Reply:
👍 read up on DTO, will attempt to ditch the DTO all together and update the pr.

Reply:
^ Yep seems like there's a bit of mix and match happening, see my comment in `backend/python/app/graphql/resolvers/entity_resolver.py`

Starter code hides the ORM from the controller layer (i.e. REST endpoints or GraphQL resolvers) by using service classes which deal with DTOs instead of the underlying model. If you'd like to keep this pattern then I would recommend not using `graphene_sqlalchemy`. If using `graphene_sqlalchemy` then I don't think the service layer (plus DTOs) will be necessary at all.

I don't know what Python + GraphQL best practices are, personally biased towards first approach for obvious reasons 😛 but totally up to your team to decide!

Reply:
ah I ended up flip flopping on this lol: DTOs are much easier to work with for things like Users where emails and passwords arent part of the ORM, but auth_id is hidden. DTOs are back in for now and `graphene_sqlalchemy` has been removed. 

I'm leaving in `/resources/models` for now as we can remove them later when we do the full REST nuke. 

Ty for the feedback!